src/sentry/api/serializers/models/release.py

```diff
@@ -251,9 +251,29 @@ def get_attrs(self, item_list, user, **kwargs):
         return result
 
     def serialize(self, obj, attrs, user, **kwargs):
+        def expose_version_info(info):
+            version = {"raw": info["version_raw"]}
+            if info["version_parsed"]:
+                version.update(
+                    {
+                        "major": info["version_parsed"]["major"],
+                        "minor": info["version_parsed"]["minor"],
+                        "patch": info["version_parsed"]["patch"],
+                        "pre": info["version_parsed"]["pre"],
+                        "buildCode": info["version_parsed"]["build_code"],
```

Comment:
Does `parse_release` guarantee that all of these attributes will be prevent in `version_parsed`?

Reply:
Yep. It always has these.